### ISSUE TITLE
Works locally but fails on server

### ISSUE THREAD

**Issue by jonno007** [jhabjan/Ghostscript.NET#31]
Hi hope you can help.  I am pretty sure its a simple problem.

I have the service running via IIS Express (within Visual Studio) however when I deploy it on my dev server with IIS 8 nothing works.

I get the following error when I try to convert a PDF to an image

> **An error occured when call to 'gsapi_new_instance' is made: -100**

I have my application running as an administrator account and I have installed the ghostscript library on the server and rebooted.

The code I use works fine locally.  Is there anything else I need to install on the server?

**Comment by jonno007** [comment]
I would like to add I added the ghostscript.net.viewer on the server and that works fine.

**Comment by JohnVLinton** [comment]
Sir, you might want to review this issue as I had it too here:
https://stackoverflow.com/questions/45214870/ghostscript-weird-lock-on-dll-an-error-occured-when-call-to-gsapi-new-instance

However that only temporarily got me to the next leg of the journey as I found an OutofMemory issue as I am deploying the .dll on a website and it does not seem to permit multiple thread access for raster.Open(fileStream, _version, true);
so I ended up using the false parameter but with a static lock around that code region.

Everything is 99% okay now for me but after several months someone had a "Module could not be loaded. at Microsoft.WinAny.Interop.DynamicNativeLibrary..ctor(String fileName)" on the server that then strangely went away and I have not been able to replicate.

So at this point I want to stay with GS but not sure yet how to fix this rare-bird event.
